pkg/noderesourcetopology/logging/logging.go

```diff
@@ -33,6 +31,7 @@ const (
 	KeyFlow          string = "flow"
 	KeyContainer     string = "container"
 	KeyContainerKind string = "kind"
+	KeyGeneration    string = "gen"
```

Comment:
or `generation`? size of log entries is a concern too

Reply:
I would prefer readability over compactness

Reply:
fair enoug, "generation" it is